src/python/pants/backend/codegen/thrift/java/thrift_defaults.py

```diff
@@ -29,6 +29,8 @@ def register_options(cls, register):
              help='The default Java namespace to generate for java_thrift_library targets.')
     register('--namespace-map', type=dict, advanced=True, default={},
              help='The default namespace map to generate for java_thrift_library targets, {old: new}.')
+    register('--compiler-args', type=list, advanced=True, default=[],
```

Comment:
Correspondingly (about 6 lines up from this comment), we should deprecate the `--rpc-style` arg ([an example usage](https://github.com/pantsbuild/pants/blob/ae1ea88d5a280120dff3a4cff2141b774ccca139/src/python/pants/option/global_options.py#L159): `removal_version` and `removal_hint`).